ui/react-components/components/CancelConfirmation/CancelConfirmation.jsx

```diff
@@ -27,11 +27,18 @@ const CancelConfirmation = (props) => {
                     <FormattedMessage id={'APPOINTMENT_CANCEL_CONFIRMATION_TITLE'} defaultMessage={'Wait!'}/>
                 </h1>
                 <div className={classNames(cancelModalBody)}>
-                    <FormattedMessage id={'APPOINTMENT_CANCEL_CONFIRMATION_TEXT'}
-                                      defaultMessage={'Are you sure you want to cancel adding the new appointment?This will erase everything you have filled. Nothing will be saved.'}/>
+                    {isEdit ? (
```

Comment:
Can we the message to be displayed be passed as a prop?? So that cancel confirmation is available for reuse?